src/components/FormLabel/FormLabel.module.scss

```diff
@@ -1,7 +1,9 @@
+.label-reset {
+  composes: base-css-reset from '../../styles/reset.scss';
+}
+
 .label {
   display: block;
```

Comment:
Why does it need to be a separate class? Why not put the `composes` right inside the `label` class?

Reply:
I find it helpful to know which properties on my component are being added purely for reset purposes vs things that are actually component styles.